### ISSUE TITLE
[1.12]CustomNPCs conflicts with Assembler

### ISSUE THREAD

**Issue by lordfriend** [BluSunrize/ImmersiveEngineering#3270]
Minecraft Version 1.12.2
Forge Version 14.23.5.2768
IE version: 0.12-86

Other mods:
```
CodeChicken Lib 3.2.2.353
CustomNPCs 1.12
Just Enough Items 4.11.0.212
```

I leave only these mods to test. We I found the Assembler from IE cannot craft most items which it can show the result item. It is known CustomNPCs doesn't add any recipes and all recipes can be craft at crafting table. After I remove CustomNPCs mod, Assembler works fine.

**Comment by BluSunrize** [comment]
Well, CustomNPCs' code is private. I can only theorize that they may do things to recipes in order to allow NPCs to craft them?

Maybe @Noppes, who I believe is the dev, can shed some light on this?

**Comment by crazycarl** [comment]
I have found that updating to Immersive Engineering version 0.12-87 fixed these issues for me.

**Comment by crazycarl** [comment]
After playing for awhile, the issue came back. It seems to be a bit inconstant.
Man, way to make me look bad.

**Comment by Brycey92** [comment]
CustomNPCs adds custom NBT data to every single item in the game, which causes issues with a LOT of mods. This is the most likely cause of the issue. Specifically ignoring this custom NBT tag compound when matching recipes for the Assembler would fix it.

**Comment by Brycey92** [comment]
I know this was last commented on over a year ago, but was it closed because it was fixed or because it won't be fixed?

**Comment by BluSunrize** [comment]
It won't be fixed, all support for 1.12 has stopped.
